client/main.go

```diff
@@ -18,7 +18,19 @@ package main
 import (
 	"flag"
 	"log"
+	"fmt"
+	"strings"
 )
+type envars []string
```

Comment:
I'm not sure this custom type is adding any significant value, since Set() is never used and String() actually mimics the default behavior of printing arrays of string with fmt.Sprintf("%v", strArray)

Reply:
I tried the approach where we have a regular string array and use it instead of the custom type, but that does not work. flag.Parse() calls the Set() method internally for Var flags so we do need a custom type.

Reply:
Any custom flag should implement the Value interface- https://pkg.go.dev/flag#Value

Reply:
I see, so I think we'll want to diverge from the docker run pattern of allowing repeated flags. It's going to make it trickier when adding it as a parameter GitHub workflows: https://github.com/GoogleCloudPlatform/functions-framework-conformance/blob/master/.github/workflows/buildpack-integration-test.yml

I think for the flow it'll be easier to have one, comma-separated string var, like `--envs=FUNCTION_TARGET=myfn,FUNCTION_CONCURRENCY=8` that we can pass straight through to the client `--envs=${{ inputs.envs }}

That's similar to how it works in `gcloud functions deploy` https://cloud.google.com/sdk/gcloud/reference/functions/deploy#--set-env-vars